### ISSUE TITLE
anmn_temp_gridded_product.py - Static max depth per site

### ISSUE THREAD

**Issue by ggalibert** [aodn/data-services#698]
It would be good to set the maximum depth of the T re-gridded product consistent accross different deployments. Say for example site nominal depth + 10%. So for a site which nominal depth is 100m, the max depth for which the product can give data would be 110m.

It is important to hard code site nominal depth somewhere and not relie on the value given in the global attributes since this can be erroneously/inconsistently documented.

**Comment by lbesnard** [comment]
How would you see this being hard coded ? should we have a text file with this information ?

```site_code = nominal_depth``` ?

**Comment by ggalibert** [comment]
> How would you see this being hard coded ? should we have a text file with this information ?
> `site_code = nominal_depth` ?

Yes, chosen `nominal_depth` could be the deepest found so far in all the deployments of a site.

**Comment by lbesnard** [comment]
I guess we can close this now @ggalibert

**Comment by ggalibert** [comment]
Not yet!

**Comment by ocehugo** [comment]
2 cents:

I don't see a reason why the 10% rule. The nominal depth of the site will never be reached by any measurement - even bottom ADCP are some meter above the floor. 

If any measurement falls into or below the nominal depth this helps indicate an error on the depth value of either the deployment or the instrument.

I think it would be a good standard to set the x(z) bottom most value to masked at z == nominal depth.

**Comment by ggalibert** [comment]
Some sites can be located on steep continental slope so sampled depth and site depth can vary across deployments. 

One feedback was that users don't want to retrieve products that have inconsistent vertical spans if requested over different time periods, hence the idea of hard coding the nominal site depth + 10% or so in order to set a consistent vertical span per site.

**Comment by ocehugo** [comment]
Shall we close!? Code is now deprecated and the task can be done by the end user by combining/subsetting the long-time series products.
 @ggalibert , @lbesnard
